docs/src/pages/components/app-bar/app-bar.md

```diff
@@ -19,6 +19,10 @@ It can transform into a contextual action bar or be used as a navbar.
 
 {{"demo": "pages/components/app-bar/ButtonAppBar.js", "bg": true}}
 
+## AppBar with responsive menu
```

Comment:
```suggestion
## App Bar with responsive menu
```

Reply:
Please move this demo below App Bar with menu

Reply:
done